abtesting/Shared/AppConfig.swift

```diff
@@ -74,16 +74,17 @@ class AppConfig: ObservableObject {
       }
     }
   #endif
-
-  @objc func printInstallationAuthToken() {
-    Installations.installations().authTokenForcingRefresh(true) { token, error in
-      if let error = error {
-        print("Error fetching token: \(error)")
-      } else if let token = token {
-        print("Installation auth token: \(token.authToken)")
+  #if !targetEnvironment(macCatalyst)
```

Comment:
Is this still needed?

Reply:
Yes, Installations is not installed due to code signing issues faced with catalyst. This is separate from CoreTelephony.

Reply:
Error example
```
2021-09-02 15:47:01.618605-0500 ABTestingExample (catalyst)[30531:354255] 8.6.1 - [Firebase/RemoteConfig][I-RCN000073] Failed to get installations token. Error : Error Domain=com.firebase.installations Code=0 "Underlying error: The operation couldn’t be completed. SecItemAdd (-34018)" UserInfo={NSLocalizedFailureReason=Underlying error: The operation couldn’t be completed. SecItemAdd (-34018), NSUnderlyingError=0x60000138cf00 {Error Domain=com.gul.keychain.ErrorDomain Code=0 "SecItemAdd (-34018)" UserInfo={NSLocalizedFailureReason=SecItemAdd (-34018)}}}.
```

Reply:
@maksymmalyhin Any ideas/suggestions here?

Reply:
Installations SDK uses Keychain to store tokens. On Catalyst we need to configure Keychain sharing to get access to Keychain.

Reply:
Probably eventually we should add comments with instructions to help developers to figure it out.

Reply:
See https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/configure_test_keychain.sh and how it gets used in the firebase-ios-sdk

Reply:
But I'm confused why we don't have a similar issue for macos testing?

Reply:
For macOS, a prompt asks for permission to allow access to Keychain, which then allows for the normal functioning of the application. Mac Catalyst doesn't surface a prompt when using the same entitlements (which don't include Keychain sharing).

Reply:
Hmm. Then the code is probably good as it is with docs since the script I pointed at makes sense for testing but not for a quickstart.